package.json

```diff
@@ -27,17 +27,17 @@
         "reset": "pnpm -r --include-workspace-root --parallel exec pnpm dlx rimraf dist node_modules"
     },
     "devDependencies": {
-        "@workleap/eslint-plugin": "workspace:*",
-        "@workleap/typescript-configs": "workspace:*",
         "@changesets/cli": "2.26.1",
         "@changesets/changelog-github": "0.4.8",
-        "typescript": "5.0.4",
-        "eslint": "8.38.0",
         "@types/node": "18.15.12",
         "@types/jest": "29.5.1",
+        "@workleap/eslint-plugin": "workspace:*",
+        "@workleap/typescript-configs": "workspace:*",
+        "eslint": "8.38.0",
         "jest": "29.5.0",
         "ts-jest": "29.1.0",
```

Comment:
is "ts-jest" still necessary? haven't checked yet, this is a personal note to not forget

Reply:
I haven't done anything related to test, it will be in my next PR!